test/TestCasesDevtoolSourceMapDebugIds.longtest.js

```diff
@@ -0,0 +1,8 @@
+const { describeCases } = require("./TestCases.template");
+
+describe("TestCases", () => {
+	describeCases({
+		name: "devtool-source-map-debugids",
+		devtool: "source-map-debugids"
+	});
+});
```

Comment:
Let's use just a simple test because it will increase the testing time a lot, but we don't need to test all possible variants with this meaning